.github/workflows/android.yml

```diff
@@ -47,8 +47,20 @@ jobs:
         if: ${{ success() && matrix.type == 'apps' }}
         run: ./android/ci_build_apps.sh
 
+      - name: Build vcpkg manager
+        if: ${{ success() && matrix.type == 'vcpkg_manager' }}
+        run: ./android/ci_build_vcpkg_manager.sh
+
+      - name: Build vcpkg libs
+        if: ${{ success() && matrix.type == 'vcpkg_libs' }}
+        run: ./android/ci_build_vcpkg_libraries.sh
+
+      - name: Build vcpkg apps
+        if: ${{ success() && matrix.type == 'vcpkg_apps' }}
+        run: ./android/ci_build_vcpkg_apps.sh
+
       - name: Deploy to BinTray
-        if: ${{ success() && ! contains(matrix.type, 'libs') }}
+        if: ${{ success() && ! contains(matrix.type, 'libs') && ! contains(matrix.type, 'vcpkg_libs') }}
```

Comment:
looks redundant:
`! contains(matrix.type, 'libs')` should already filter out both plans that contains `libs` it their names

Reply:
fix